### DISCUSSION TITLE
Projeto com dados de Vacinação - COVID19

### DISCUSSION THREAD

**Discussion by alicescfernandes** [dssg-pt/covid19pt-data#522] [Montra de Projectos]
Bons dias. Espero que esta seja a maneira correta de partilhar estes projetos!
Já a alguns dias que ando a desenvolver um site que mostra os números atuais no que diz respeito à vacinação para a COVID19. Inicialmente era para ter mais algumas páginas, mas por enquanto, queria ver se fazia nascer mais alguns gráficos que pudessem ser pertinentes no contexto das vacinas.

O site é este http://vacinacaocovid19.pt/ e o código fonte é aberto (https://github.com/alicescfernandes/mapa-vacinacao-c19). 

Os dados vêm de dois sitios diferentes: 
- O numero total de vacinas administradas é retirado do dashboard da DGS
- O numero total das doses (1ª e 2ª)  administradas são retirados da comunicação oficial da DGS. Estes dados (parece-me que) costumavam ser distribuídos no mesmo _endpoint_ que o do dashboard, mas parece que desde deixaram de ser divulgados dessa maneira.

Gostaria de ter o vosso _feedback_, e se alguém tiver ideias de gráficos que gostaria de ver, sintam-se livres de contribuir para o projeto! Pessoalmente, gostaria de ver um com o numero de vacinas disponíveis, mas ainda não consegui arranjar maneira consistente de obter esses números!

PS: Ainda tenho de trabalhar no readme do repositório!

**Comment by ishouldbedany** [comment]
Olá @alicescfernandes,

Muito bom aspecto! O projecto da Pandemia Clara tem uma página similar: https://pandemiaclara.sapo.pt/plano-de-vacinacao/, mas a tua tem um foco mais analítico (e com uma API :clap: ). 

Embora seja informação tremendamente útil, não creio que o número de vacinas disponíveis esteja disponível numa fonte fixa e fiável: são mais números largados numa entrevista aqui, numa conferência de imprensa ali...  :sleeping: 

Também estamos a integrar dados de vacinação aqui no repositório e temos algunas discussões a decorrer acerca deles, nomeadamente no #518 e #501. Talvez alguma informação lá contida seja útil. 

Parabéns mais uma vez.

**Reply by alicescfernandes** [reply]
Pois, era o que estava a temer... Era muito fixe ter esses dados. A ideia é fazer crescer numero de gráficos, e já tenho algumas ideias e sugestões de pessoas que me foram dando feedback. 

Nos próximos dias o site vai sofrer updates em relação a isto. Estou a tentar não ter informação duplicada (daí referenciar o Pandemia Clara no _header_) e focar-me apenas nestes números da vacinação, e enquadrar estes dados em várias prespetivas.  A API tinha alguns problemas, mas agora já dá para ser consumida, e também os dados são _open source_ (para ver se realmente conseguimos realmente puxar pela transparência das entidades responsáveis)

Obrigado pelo feedback!

**Comment by joserodpt** [comment]
Olá alice!
Gostava de perguntar se não há problema em integrar os dados da tua API na minha app.
Obrigado!

**Reply by alicescfernandes** [reply]
Força! E depois (se quiseres claro) mostra a app!

**Reply by joserodpt** [reply]
Antes disso gostava de falar contigo, se possível!
Podemos falar por onde? Email?

**Reply by alicescfernandes** [reply]
Podemos sim!
